src/index.js

```diff
@@ -4,7 +4,13 @@ import insertStyle, { removeStyle, getStyle } from './insert-style';
 
 const cache = new Map();
 
-export default function withStyles(css) {
+export default function (css, values) {
+
+  if (Array.isArray(css)) {
+    // Forward string literals to csjs
+    css = csjs(css, values);
```

Comment:
Instead of modifying the `css` param, can we rename it in the signature and then use `css` locally here?